src/commands/yogalog.js

```diff
@@ -0,0 +1,20 @@
+import { SlashCommandBuilder } from "discord.js";
+import { getXataClient } from "./../xata.js";
+
+export default {
+  data: new SlashCommandBuilder()
+    .setName('yogalog')
+    .setDescription('Lists all your yoga sessions'),
+  async execute(interaction, xata = getXataClient()) {
```

Comment:
**question(non-blocking):** is this the same as `execute: (interaction, xata = getXataClient()) => {}`?

Reply:
I saw they do this in the docs, just very unfamiliar to with a function declaration with await in the middle of a object declaration.

Reply:
Hm, I am not sure which syntax is more expressive, but I had to do this to use `await` inside it.

Reply:
I forgot the async in my question: `execute: async (interaction, xata = getXataClient()) => {}`.

Reply:
Hm, other default keys work that way - but functions? Not sure.